### PR TITLE
node_boot_time metric not available anymore

### PR DESCRIPTION
**Pull request by eye0fra**
1. node_boot_time metric is not anymore exported by the prometheus node exporter. Using node_exporter_build_info instead.
2. Put multi and includeAll to false

### CODE REVIEW

File: roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json [hunk 1]
```diff
@@ -19425,12 +19425,12 @@
 			"current": {},
 			"datasource": "##DS_PR##",
 			"hide": 0,
-			"includeAll": true,
+			"includeAll": false,
```
[the remark this answers is not in the document]
Comment (eye0fra) [reply roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json]: I noticed that most of the graph don't work with selector all.

File: roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json [hunk 2]
```diff
@@ -19425,12 +19425,12 @@
...
 			"label": "Host:",
-			"multi": true,
+			"multi": false,
```
[the remark this answers is not in the document]
Comment (eye0fra) [reply roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json]: I noticed that most of the graph don't work with multi graph.

File: roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json [hunk 3]
```diff
@@ -19425,12 +19425,12 @@
...
 			"name": "node",
 			"options": [],
-			"query": "label_values(node_boot_time{job=\"kubernetes-nodes-exporter\"}, instance)",
+			"query": "label_values(node_exporter_build_info{job=\"kubernetes-nodes-exporter\"}, instance)",
```
[the remark this answers is not in the document]
Comment (eye0fra) [reply roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json]: Ah ok, make sense ;)

File: roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrsiano) [reply roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json]: some of them, but anyway we need it for future usage. and for the graphs that dose use it.

[the remark this answers is not in the document]
Comment (eye0fra) [reply roles/openshift_grafana/files/dashboards/node-exporter-full-dashboard.json]: ok I will keep the change just for the node_boot_time.

**Comment by vrutkovs** [comment]
3.9 install is broken, merging manually

**Pull request merged**